h2o-core/src/main/java/water/FailedNodeWatchdogExtension.java

```diff
@@ -24,7 +26,15 @@ public void printHelp() {
                         "    -watchdog_client\n" +
                         "          Same as the client except the that cluster is stopped when this client \n" +
                         "          disconnects from the rest of the cloud or the cloud is stopped when it doesn't \n" +
-                        "          hear heartbeat from the client for specified amount of time."
+                        "          hear heartbeat from the client for specified amount of time. \n" +
+                        "    -watchdog_client_connect_timeout\n" +
```

Comment:
`watchdog_client_connect_timeout` - when client is already connected. This timeout is used to track if we lost the client
`watchdog_stop_without_client` - when we are waiting for the client initially to connect.

The watchdog works in a way that it waits for the watchdog client to connect and then the nodes start tracking if they see the client. However if the client never connects ( for example because it died during start up ) the cluster remains alive ( not desired behaviour in watchdog mode )

The second property is used to handle this case